module/VuFind/src/VuFind/Controller/AbstractRecord.php

```diff
@@ -955,8 +957,12 @@ protected function showTab($tab, $ajax = false)
         $view->backgroundTabs = $this->getBackgroundTabs();
         $view->tabsExtraScripts = $this->getTabsExtraScripts($view->tabs);
         $view->loadInitialTabWithAjax
-            = isset($config->Site->loadInitialTabWithAjax)
-            ? (bool)$config->Site->loadInitialTabWithAjax : false;
+            = isset($config->Site->loadInitialTabWithAjax) && (bool)$config->Site->loadInitialTabWithAjax;
```

Comment:
An even simpler version might be:

`$view->loadInitialTabWithAjax = (bool)($config->Site->loadInitialTabWithAjax ?? false);`